### PR TITLE
Add format js in render_wizard for javascript support

### PR DESCRIPTION
**Pull request by sirion1987**
Some change in render_redirect.rb for js support

### CODE REVIEW

File: lib/wicked/controller/concerns/render_redirect.rb [hunk 1]
```diff
@@ -6,7 +6,10 @@ def render_wizard(resource = nil, options = {})
     process_resource!(resource)
 
     if @skip_to
-      redirect_to wizard_path(@skip_to, @wicked_redirect_params || {}), options
+      respond_to do |format|
+        format.html { redirect_to wizard_path(@skip_to, @wicked_redirect_params || {}), options }
+	format.js   { render wizard_path(@skip_to, @wicked_redirect_params || {}), options }
```
Comment (schneems) [inline lib/wicked/controller/concerns/render_redirect.rb]: indentation is different on this line

**Comment by schneems** [comment]
What does the client do in this case? Looks like this change would render the whole page. At that point the client would be re-painting the whole view, why not just do an actual page load and you don't have to worry about URL history state or anything.

**Comment by sirion1987** [comment]
I wrote a [guide](http://stackoverflow.com/questions/39367983/rails-wicked-wizard-with-javascript/39375038#39375038) on stackoverflow for this problem. The problem comes when you reach the second step. In this case, for example, the url form is like "wizard_path(:second_step)". 

I would like to make available the render_wizard @object method for the other formats. For js format,  I have to necessarily do the render @skip_to ( alias @next_step )

**Comment by kevin-jj** [comment]
@schneems  In ecommerce platform,  one-page checkout will boost the conversion rate according to some survey, however, wicked gem is best way to implement one-page.

**Comment by schneems** [comment]
I wrote up a long reply on this many days ago and apparently didn't submit. 

The answer on stack overflow is at zero points. I don't think this direction forward of responding to JS is something all apps can take advantage of. I think this would be better as a stand alone gem `wicked-js` or something along those lines. If we get some more people using it in production and all giving feedback we can consider porting it to be an official wicked feature. In the mean time it still allows you and anyone else who wants to use that ability.

**Pull request closed without merging**